ci/build_libtelio.py

```diff
@@ -317,7 +329,71 @@ def main() -> None:
         assert False, f"command '{args.command}' not supported"
 
 
+def try_download_artifacts(
+    tag_prefix, path_to_save, target_arch, target_os, moose=False
+):
+    def get_token():
+        if "LLH_GROUP_TOKEN_FLAKY_TESTS" in os.environ:
```

Comment:
Maybe it would be better either to create dedicated token environment variable for fetching artifacts, or rename this one? 

It my be somewhat surprising, if `FLAKY_TESTS` token is changes/renewed/updated etc. and something happens to artifact downloading.

Reply:
Fixed